contracts/bepro/Network_Registry.sol

```diff
@@ -90,12 +116,56 @@ contract Network_Registry is ReentrancyGuardOptimized, Governed {
 
     function changeAmountForNetworkCreation(uint256 newAmount) external payable onlyGovernor {
         require(newAmount > 0, "C1");
-        lockAmountForNetworkCreation = newAmount * 10 ** 18;
+        lockAmountForNetworkCreation = newAmount;
     }
 
     function changeLockPercentageFee(uint256 newAmount) external payable onlyGovernor {
         require(newAmount.div(10000) <= 10, "CLF1");
         lockFeePercentage = newAmount;
     }
 
+    function changeGlobalFees(uint256 _closeFee, uint256 _cancelFee) external payable onlyGovernor {
+        require(_cancelFee >= 0, "CGF1");
+        require(_closeFee >= 0, "CGF1");
+        closeFee = _closeFee;
+        cancelFee = _cancelFee;
+        emit ChangedFee(closeFee, cancelFee);
+    }
+
+    function addAllowedTokens(address[] memory _erc20, bool transactional) public payable {
+        mapping(address => address) storage pointer = transactional ? allowedTransactionalTokens : allowedRewardTokens;
+        address[] storage array = transactional ? allowedTokens[0] : allowedTokens[1];
```

Comment:
Overcomplicated code. A simple setter is easier .

Reply:
@hvasconcelos https://github.com/taikai/dappkit/pull/44#discussion_r928624734